lib/Mojolicious/Plugin/PODViewer.pm

```diff
@@ -274,16 +288,41 @@ __DATA__
 @@ layouts/podviewer.html.ep
 <!DOCTYPE html>
 <html>
-    <head>
-        <title><%= title %></title>
-    </head>
-    <body>
+  <head>
+    <title><%= title %></title>
+    %= tag 'meta', name => 'viewport', content => 'width=device-width, initial-scale=1, shrink-to-fit=no'
+    <%= stylesheet 'https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css',
+        integrity => 'sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS',
+        crossorigin =>'anonymous' %>
+    %= stylesheet begin
+      .codesample {
+        background-color: #f8f9fa;
+        padding: 1em;
+      }
+      .more {
+        font-size: 0.5em;
+        font-weight: 50;
+      }
+      .permalink {
+        font-size: 50%;
+      }
+    % end
+
+  </head>
+  <body>
+    <nav class="navbar navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">PODViewer</a>
+    </nav>
+
+    <div class="container">
     %= content
-    </body>
+    </div>
+    %= javascript '/mojo/prettify/run_prettify.js'
+  </body>
 </html>
 
 @@ podviewer/perldoc.html.ep
-<div class="crumbs">
+<h1 class="crumbs mt-3 mb-3">
```

Comment:
The crumb trail can't be the title of the document, so it can't be an `<h1>` tag. If someone wants to make it big, they're free to, but not by default. This should remain a `<div>`.